Entities/Items/Crate/Crate.as

```diff
@@ -184,8 +184,7 @@ void Land(CBlob@ this)
 
 bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 {
-	return (this.getName() == blob.getName())
-	    || ((blob.getShape().isStatic() || blob.hasTag("player") || blob.hasTag("projectile")) && !blob.hasTag("parachute"));
+	return (blob.getShape().isStatic() || blob.getPlayer() !is null || blob.hasTag("projectile")) && !blob.hasTag("parachute");
```

Comment:
this prevents crates from colliding with each other again

Reply:
fixed